.github/workflows/10-alpine3.10.yml

```diff
@@ -0,0 +1,23 @@
+name: 10 on alpine3.10
+
+on:
+  push:
+    paths:
+      - functions.sh
+      - test-build.sh
+      - test-image.bats
+      - 10/alpine3.10/Dockerfile
+  pull_request:
+    paths:
+      - functions.sh
+      - test-build.sh
+      - test-image.bats
+      - 10/alpine3.10/Dockerfile
+
+jobs:
+  build:
```

Comment:
Seems this name is used in the status? Can we set a display name here?

Reply:
The job is the middle part of the display `10 on alpine3.10 / build (pull_request)`. Th start of the display comes from the `name` above

Reply:
Not on mobile

https://drive.google.com/file/d/1eF3hdszKvhtxdGuoKg8qJK41fK64xoCK/view?usp=drivesdk

(No idea how to upload screenshot from mobile, sorry about the google drive link)

Reply:
On a computer now, so can paste it

![image](https://user-images.githubusercontent.com/1404810/79908321-51d33100-841b-11ea-88c3-14f1c939e62c.png)

Reply:
This is still an issue. @nschonni if you give the job a `name` it works on mobile as well

Reply:
I can't see it on mine, but i've duplicated the root name again

Reply:
works now 👍